src/Switch-FTDI_FTD2XX/main.py

```diff
@@ -0,0 +1,143 @@
+# This Device Class is published under the terms of the MIT License.
+# Required Third Party Libraries, which are included in the Device Class
+# package for convenience purposes, may have a different license. You can
+# find those in the corresponding folders or contact the maintainer.
+#
+# MIT License
+#
+# Copyright (c) 2024 SweepMe! GmbH (sweep-me.net)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+# SweepMe! device class
+# Device: Arduino MCP4728
+
+import os
+import sys
+
+file_dir = os.path.dirname(__file__)
+sys.path.append(file_dir)
+
+libs_path = file_dir + os.sep + "libs"
+libraries_path = file_dir + os.sep + r"libraries\libs_39_64"
+sys.path.append(libraries_path)
+
+
+import binascii
+
+import ftd2xx as ftd
+from pysweepme import EmptyDevice
+
+
+class Device(EmptyDevice):
+    description = """
+        Need FTDI driver installed
+        Can send initial command when initializing or end command when deinitializing
+    """
+
+    def __init__(self) -> None:
+        EmptyDevice.__init__(self)
+
+        self.shortname = "FTDI245"
+        self.variables = []
+        self.units = []
+
+        self.port_manager = False
+        self.port_types = ["USB"]
```

Comment:
The port types are used by the port manager to automatically add ports to the list of ports after clicking the button "Find ports" without doing this in an "find_ports" method of the driver. The port type "USB" refers to "USBTMC" which is not related to the USB port used by the FTDI chip. So this line can be removed here.